README.md

```diff
@@ -14,15 +14,19 @@ const config = {
   amqp: { url: 'amqp://user:pw@myserver/blah', exchange: 'myexchange', routingKey: 'keyToRouteTo' }
 };
 const Logger = require('log2amqp')(config);
-let logger = Logger();
+let logger = Logger(loggerConfig);
```

Comment:
so what is `loggerConfig` vs `config`? why do we have 2 levels of config? is one for defaults and the other to be overridden when we create the logger? also should it be `logger = new Logger(loggerConfig)` or should it be something like `logger = log2amqp(loggerConfig)`?

Reply:
looking at the code and how you have used it in the api it looks like you didn't actually need to change this from what we had before?

Reply:
I have removed the doco that was left there unintentionally.